backend/ops_api/ops/utils/cans.py

```diff
@@ -1,43 +1,63 @@
 from typing import Optional, TypedDict
 
-from models.cans import CAN, BudgetLineItem, BudgetLineItemStatus, CANFiscalYear, CANFiscalYearCarryOver
+from models.cans import CAN, BudgetLineItem, BudgetLineItemStatus, CANFiscalYear, CANFiscalYearCarryForward
 
 
 class CanFundingSummary(TypedDict):
     """Dict type hint for total finding"""
 
     can: CAN
-    current_funding: float
+    received_funding: float
     expected_funding: float
     total_funding: float
+    carry_forward_funding: float
+    carry_forward_label: str
     planned_funding: float
     obligated_funding: float
     in_execution_funding: float
-    available_funding: float
+    available_funding: str
```

Comment:
Curious why this is a `str`.

Reply:
I am not sure, we are doing formatting on the server side. I don't think that's what we should be doing. I can adjust it and make sure things work their way through correctly on the front-end.